notifications_utils/recipient_validation/errors.py

```diff
@@ -23,7 +25,10 @@ class Codes(StrEnum):
 
     # TODO: Move this somewhere else maybe? Maybe not?
     ERROR_MESSAGES = {
-        Codes.INVALID_NUMBER: "Not a valid phone number",
+        # this catches numbers with the right length but wrong digits
+        # for example UK numbers cannot start "06" as that hasn't been assigned to a purpose by ofcom,
+        # or a 9 digit UK number that does not start "01" or "0800".
+        Codes.INVALID_NUMBER: "TODO: CONTENT! Number is not valid – double check the mobile number you entered",
```

Comment:
we could give this a content that's not TODO, and move TODO to comment to look at later.

Reply:
done